libbpf-rs/src/ringbuf.rs

```diff
@@ -35,6 +35,7 @@ pub struct RingBufferBuilder<'a> {
 }
 
 impl<'a> RingBufferBuilder<'a> {
+    /// Create a new `RingBufferBuilder` not containing any callbacks.
```

Comment:
"not containing any callbacks" seems like an unnecessary part? Builder is not a final RingBuffer handler, so obviously it starts with essentially empty set callbacks?

Reply:
I removed this part. Thanks for taking a look.